src/index.tsx

```diff
@@ -119,7 +130,7 @@ export function useSpeechRecognition(options: SpeechRecognitionOptions = default
       );
 
       if (options.onResult) {
-        options.onResult(final, interim);
+        options.onResult(final, interim, event);
```

Comment:
I think passing the raw event to the onResult handler is helpful, since we don't actually know if `final` and `interim` will cover all the use cases. Sometimes we need metadata from the raw event